src/pages/completePledge.js

```diff
@@ -37,8 +36,7 @@ class CompletePledgePage extends React.Component {
   };
 
   async componentDidMount() {
-    const { getLoggedInUser } = this.props;
```

Comment:
If component gets updated later (which will happen because `LoggedInUser` is fetched after the mount) then we won't get the updated `LoggedInUser`. We don't need to store the user in the state anymore, we can rely on `this.props` directly.